webapp/ansible/roles/service-vm/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+  - name: Fix Debian locale error.
+    command: update-locale LANGUAGE="en_US.UTF-8" LC_ALL="en_US.UTF-8" #"echo 'LANGUAGE="en_US.UTF-8"' > /etc/default/locale && echo 'LC_ALL="en_US.UTF-8"' >> /etc/default/locale # The first redirection intentionally creates the file.
+
+  - name: Upgrade packages.
+    apt: upgrade=dist update_cache=yes
+
+  - name: Install python-pip, python-dev, sudo and git packages.
+    apt: name={{ item.name }} state=latest
+    with_items:
+      - { name: 'python-pip' }
+      - { name: 'python-dev' }
+      - { name: 'sudo'       }
+      - { name: 'git'        }
```

Comment:
You may rewrite this as:

```
- name: Install python-pip, python-dev, sudo and git packages.
  apt: name={{ item }} state=latest
  with_items:
    - python-pip
    - python-dev
    - sudo
    - git
```

which makes task a bit cleaner to read. If, however, you plan to make this more specific (i.e. define explicitly version numbers instead of `latest` then leave it as is.